contrib/chafa/template.py

```diff
@@ -0,0 +1,48 @@
+pkgname = "chafa"
+pkgver = "1.14.0"
+pkgrel = 0
+build_style = "configure"
```

Comment:
this is gnu_configure

Reply:
no, doing so trigger a secondary configure and fails.

Reply:
autogen.sh calls configure at the very with all extra args https://github.com/hpjansson/chafa/blob/b77c28c85bcae5f7c5f42f1c820ac73393152865/autogen.sh#L110
doing a gnu_configure where the configure is skipped would boil down to the same thing

Reply:
> autogen.sh calls configure at the very end with all extra args

the thing you linked to has a `NOCONFIGURE` right there

> where the configure is skipped would boil down to the same thing  

except for the heaps of extra options that gnu_configure passes compared to configure

Reply:
fair enough